src/Types.jl

```diff
@@ -511,7 +511,9 @@ function handle_repos_develop!(ctx::Context, pkgs::AbstractVector{PackageSpec},
                     # Relative paths are given relative pwd() so we
```

Comment:
We can also change this, such that when deving a path you write the path relative the project file, but that will be very annoying I think.

Reply:
Yes